src/org/intellij/erlang/psi/impl/ErlangPsiImplUtil.java

```diff
@@ -1082,7 +1082,7 @@ public int compare(@NotNull VirtualFile o1, @NotNull VirtualFile o2) {
     return behaviours;
   }
 
-  private static void addDeclaredBehaviourModuleNames(@NotNull ErlangFile file, @NotNull Set<String> behaviourNames) {
+  public static void addDeclaredBehaviourModuleNames(@NotNull ErlangFile file, @NotNull Set<String> behaviourNames) {
```

Comment:
You've made this method public to implement ```getDeclaredBehaviourPaths```.

Also, your changes rendered ```org.intellij.erlang.psi.impl.ErlangPsiImplUtil#getImplementedBehaviourModuleNames``` and ```org.intellij.erlang.psi.impl.ErlangPsiImplUtil#getAppliedParseTransformModuleNames``` unused. Consider removing them.